app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
```

Comment:
toán tử ||= lấy giá trị của @current_user nếu current_user != nil, nếu current_user == nil thì sẽ nhận giá trị trả về của User.find_by(id: session[:user_id])